### PR TITLE
0.2.0

### PR DESCRIPTION
**Pull request by shanebo**

### CODE REVIEW

File: lib/swap.js [hunk 1]
```diff
@@ -232,7 +235,23 @@ swap.setConfirmation = (name, value) => {
...
+  swap.event('click', swap.opts.confirmCancelTrigger, closeConfirm);
+  swap.event('click', swap.opts.confirmOkTrigger, okConfirm);
+  swap.event('click', swap.opts.link, swap.click);
+  swap.event('click', swap.opts.button, swap.submit);
+  swap.event('click', swap.opts.continueTrigger, addContinueDirective);
+  swap.event('click', swap.opts.repeatTrigger, addRepeatDirective);
+  swap.event('input', swap.opts.paneForms, checkForm);
+  swap.event('submit', swap.opts.form, swap.submit);
+  swap.event('click', swap.opts.paneCloseBtn, swap.closePane);
+  swap.event('click', `.${swap.opts.paneIsOpen}`, clickOffPane);
```
Comment (joeosburn) [inline lib/swap.js:253]: ❤️

File: lib/page.js [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+import { loadSwapStyles } from './render.js';
 import { hasPane, loadPane, closePanes } from './pane.js';
 
 
+const initPage = (e) => {
+  loadSwapStyles();
+  initState();
+  fireRoutes('on', location.href, null);
+
+  if (/pane=/.test(location.hash)) {
```
Comment (joeosburn) [inline lib/page.js:14]: Should we be using `hasPane()` here?

**Review by joeosburn — APPROVED** [review]
Great work.

**Comment by shanebo** [comment]
@joeosburn our swap tests are failing now.

**Comment by joeosburn** [comment]
this is fixed @shanebo

**Comment by shanebo** [comment]
@joeosburn looks good. You have to approve this PR since I opened it.

**Review by joeosburn — APPROVED** [review]

**Comment by joeosburn** [comment]
Done!

**Pull request merged**